packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -162,19 +196,25 @@ export const rpcURLs: { [chainId: number]: string } = {
   // L1
   [ChainId.Ethereum]: loadEnvironmentVariableWithFallback({
     env: process.env.NEXT_PUBLIC_ETHEREUM_RPC_URL,
```

Comment:
To utilize this in prod, we need to make sure `NEXT_PUBLIC_ETHEREUM_RPC_URL` and `NEXT_PUBLIC_SEPOLIA_RPC_URL`  aren't set. 

We could set these as fallback instead, but it could potentially lead to breaking changes.

Reply:
This may be an insignificant concern tho and we could make it a fallback, happy to hear more opinions